app/controllers/application_controller.rb

```diff
@@ -99,4 +99,14 @@ def after_sign_in_path_for(_)
     dashboard_path
   end
 
+  # used to deliver notification on various events
+  def notify_users(action, object_type, object_id, victims)
+    Notification.create(
+          actor: current_user,
+          action: action,
+          object_type: object_type,
+          object_id: object_id,
+          victims: victims
+        )
+  end
 end
```

Comment:
Can we pass action as readable strings like "project_comment", "project_follow", "user_follow" etc? We can pass those strings as action argument and then use it to decide which action number do we need.
Would it be helpful in making actions more readable?